src/main/kotlin/gg/flyte/christmas/donate/DonateEvent.kt

```diff
@@ -0,0 +1,23 @@
+package gg.flyte.christmas.donate
+
+import org.bukkit.event.Event
+import org.bukkit.event.HandlerList
+
+/**
+ * Represents a donation event which contains details about a specific donation.
+ *
+ * @property donorName The name of the donor.
+ * @property comment Any comment made by the donor.
+ * @property time The time the donation was made.
```

Comment:
In what format?

Reply:
If you're talking about the time, that's formatted in ISO8601